spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -110,6 +115,31 @@
     }
   }
 
+  /** Class to contain venue sample data. */
+  static class Venue {
```

Comment:
I'm ok w/ this for the sample, but best practice is implementing equals, hashcode, and clone.  
(and yes, I rarely do that) See effective Java for details (I'll send an internal link)